config/args.js

```diff
@@ -9,6 +9,12 @@ const optionDefinitions = [
     alias: 'e',
     type: String,
     defaultValue: 'production'
+  },
+  {
+    name: 'tenant',
+    alias: 't',
+    type: String,
+    defaultValue: 'seek'
```

Comment:
I'm happy to not have a default.  Consuming apps would need to handle `SKU_TENANT` being undefined, but on reflection having to handle a case of `seek` is probably worse.